charts/victoria-metrics-gateway/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Victoria Metrics Gateway - Auth & Rate-Limitting proxy for Victoria Metrics
 name: victoria-metrics-gateway
 version: 0.1.49
-appVersion: v1.94.0
+appVersion: v1.94.1
```

Comment:
Seems like `appVersion` was accidentally increased instead of version, could you update the `version`?

Reply:
Sorry, following up...

Reply:
Please see https://github.com/VictoriaMetrics/helm-charts/pull/735